pkgs/top-level/python-packages.nix

```diff
@@ -1108,8 +1108,6 @@ self: super: with self; {
 
   backports-shutil-which = callPackage ../development/python-modules/backports-shutil-which { };
 
-  backports_ssl_match_hostname = callPackage ../development/python-modules/backports_ssl_match_hostname { };
```

Comment:
Nitpick: is it worth adding a throw in the aliases file?

Reply:
No, because there is no python version that could use it.